src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -63,14 +63,25 @@ const CreateProject = createReactClass({
 
   onTeamCreated() {
     let {router} = this.context;
-
     // After team gets created we need to force OrganizationContext to basically remount
     router.replace({
       pathname: router.location.pathname,
       state: 'refresh',
     });
   },
 
+  navigateNextUrl(router, docsUrl, surveyUrl) {
+    let {experiments} = this.getOrganization();
+    let showSurvey = experiments.OnboardingSurveyExperiment === 1;
+    router.push(showSurvey ? surveyUrl : docsUrl);
+
+    this.setState({inFlight: false});
+  },
+
+  getSurveyUrl({slug, projectSlug, platform}) {
+    return `/onboarding/${slug}/${projectSlug}/${platform}/survey/`;
```

Comment:
ideally we want to show this only for the first org/project they create (looks like onboarding is per project). maybe what we can do is - if create project succeeds, make another API call to check if it was the first project, and use the hook URL accordingly